.github/stale.yml

```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 30
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
```

Comment:
I am not sure with labels here.
so.... let's suppose:
when a bug or new feature comes. it will without any label?
then
if no updates for 30 days, this bot is expected to close it.
if it valuable and we are going to implement the issue, only pinned and security able to skip the check?
seems we using as `enhancement` for now.

Reply:
ref: https://github.com/probot/stale/issues/248

Reply:
ref: https://github.com/probot/stale/issues/248 they using `pinned`

Reply:
这个文件是直接用的stale bot的默认配置